### ISSUE TITLE
Casperjs is displaying blank webpage

### ISSUE THREAD

**Issue by SujathaKanathala** [casperjs/casperjs#1783]
Casperjs is displaying blank webpage and getting HTML source code using getHTML.
casperjs version is 1.1.4
phantomjs version is 2.1.1

**Comment by jefleponot** [comment]
@SujathaKanathala 
Thank for contribution it is strange that tests with HTML passed ....

I will have a look

**Comment by jefleponot** [comment]
It is working for me. Could you send your code , please ?

**Comment by kensoh** [comment]
Adding on, getHTML also works for me. The only recent situation where I have problem with blank page is after trying to log on to Mastodon all I get when driven using PhantomJS is a blank dark grey page. However when driven through SlimerJS the automation can work. So not sure if it is a PhantomJS-isolated issue. I'm not aware that Mastodon explicitly blocks PhantomJS agents.

This is the URL of Mastodon (it is a new open-source social network) - https://mastodon.social/

This is the issue I raised at PhantomJS side. But I've closed it after some time, I think PhantomJS (or new maintainer) has other more important things to think about now. https://github.com/ariya/phantomjs/issues/14975
